tests/api/api/test_artifacts.py

```diff
@@ -1008,6 +1008,47 @@ def test_list_artifacts_with_time_filters(db: Session, unversioned_client: TestC
     assert len(artifacts) == 0, "since now filter returned artifacts unexpectedly"
 
 
+@pytest.mark.parametrize("limit", [None, 6])
+def test_list_artifacts_returns_elements_by_order_updated_field(
+    db: Session, unversioned_client: TestClient, limit
+):
+    _create_project(unversioned_client)
+
+    # Create artifacts
+    number_of_artifacts = 10
+    for counter in range(number_of_artifacts):
+        artifact_name = f"artifact-{counter}"
+        json_data = _generate_artifact_body(key=artifact_name)
+        resp = unversioned_client.put(
+            STORE_API_ARTIFACTS_V2_PATH.format(project=PROJECT) + f"/{artifact_name}",
+            json=json_data,
+        )
+        assert resp.status_code == HTTPStatus.OK.value
+
+    # Make the request
```

Comment:
It's 'Do the request', but anyway you can delete the comment because it's redundant